os_stub/debuglib/debuglib.c

```diff
@@ -38,8 +38,10 @@ void libspdm_debug_assert(const char *file_name, size_t line_number, const char
 #if defined(_MSC_EXTENSIONS)
     __debugbreak();
 #endif
-#if defined(__GNUC__)
+#if defined(__GNUC__) && defined(__x86_64__)
     __asm__ __volatile__ ("int $3");
+#elif defined(__GNUC__)
```

Comment:
I would combine `_MSC_EXTENSIONS` as part of the `#if - #elif` chain, and just have the `#else` be
>Breakpoint asserts are not available on this architecture and/or compiler

Reply:
Done